hw/ip/otp_ctrl/data/otp_ctrl_img_raw.hjson

```diff
@@ -13,16 +13,6 @@
 
     // The partition and item names must correspond with the OTP memory map.
     partitions: [
-        {
-            name:  "SECRET1",
-            lock:  "False",
-            items: [
-                {
-                    name:  "FLASH_DATA_KEY_SEED",
-                    value: "<random>",
-                }
-            ],
-        }
```

Comment:
@tjaychen Do you happen to know why we initialized this here in RAW?

Reply:
I am reverting this for now so that nothing breaks.
Will file an issue here: https://github.com/lowRISC/opentitan/issues/15351